setup/bootstrap.sh

```diff
@@ -32,9 +43,8 @@ if [ -z "$TAG" ]; then
 		echo "to Ubuntu 18.04."
 		echo ""
 		TAG=v0.30
-
 	else
-		echo "This script must be run on a system running Ubuntu 18.04 or Ubuntu 14.04."
+		echo "This script must be run on a system running Ubuntu LTS versions 20.04, 18.04, or 14.04"
```

Comment:
Ubuntu 14.04 is [end of life](https://wiki.ubuntu.com/Releases) and is only available via Extended Security Maintenance (ESM).